documentation/user_doc.md

```diff
@@ -0,0 +1,55 @@
+# Nexus Replication Plugin
+
+[https://github.com/griddynamics/nexus-replication-plugin](https://github.com/griddynamics/nexus-replication-plugin)
+
+This is Nexus plugin that provides replication logic when new (Maven) artifacts are stored in a hosted repository. 
+Basically it POST an HTTP requests from master to peers instance when new artifact is saved then peers invoke artifact pooling into proxy repository.
+
+## Setting up plugin
+
+Plugin should be installed into master and peer Nexus servers both.
```

Comment:
Since for plugin there is no such thing as Master instance, I'd mention:
> You may have multiple Nexus instances configured as Masters. They will poll artifacts from each other.

I'm not sure whether Master is even a correct term here..